### DISCUSSION TITLE
Failed compile source code in Eclipse

### DISCUSSION THREAD

**Discussion by codeRemark** [dbeaver/dbeaver#13060] [DBeaver Development] [answered]
#### System information: 
- Windows 10
- DBeaver source code from github 
- Java 11
 
 
#### Describe the problem you're observing:
1） I follow the guide form  https://github.com/dbeaver/dbeaver/wiki/Develop-in-Eclipse, 
but while I import whole source into Eclipse, Eclipse report over 100 compile error ,
most relative to pom.xml , eg:
`Description	Resource	Path	Location	Type
Plugin execution not covered by lifecycle configuration: org.apache.maven.plugins:maven-clean-plugin:2.5:clean (execution: default-clean-1, phase: initialize)	pom.xml	/org.jkiss.dbeaver.ce.git.updateSite	line 8	Maven Project Build Lifecycle Mapping Problem

`

I am not sure which step wrong because I follow the guide step by step.  

again, while I run 'maven install' , it works fine. 

 please check the screensnaphot if possible, thanks for your help!

<img width="938" alt="error" src="https://user-images.githubusercontent.com/45863785/120648149-e37df080-c4ad-11eb-8b5e-3c1feb38261a.png">

**Comment by gitxpy** [comment, marked as answer]
been a while, but in case someone still wondering, these errors can be configured in eclipse  preferences>maven> ... and changed to warnings.  (same as mentionned in point 15 of the wiki page you referenced)

more details for example here : https://groups.google.com/g/vertx/c/91VC93ju1ic?pli=1
